### PR TITLE
update checkout query

### PR DESCRIPTION
**Pull request by richardmonette**
~~Looks like the storefront API moved around a few params, fixing tests.~~

cc @Shopify/gaming

### CODE REVIEW

File: Assets/Editor/TestDefaultCheckoutQueries.cs [hunk 1]
```diff
@@ -138,11 +138,11 @@ public class TestDefaultCheckoutQueries {
             string checkoutId = "an-id";
 
             var billingAddress = new MailingAddressInput("123 Test Street", "456", "Toronto", "Shopify", "Canada", "First", "Last", "1234567890", "Ontario", "A1B2C3");
-            var tokenizedPaymentInput = new TokenizedPaymentInput(new decimal(1), billingAddress, "unique_id", "some_utf8_data_string", "apple_pay");
+            var tokenizedPaymentInput = new TokenizedPaymentInput(new decimal(1), "unique_id", billingAddress, "some_utf8_data_string", "apple_pay");
```
Comment (dengjeffrey) [inline Assets/Editor/TestDefaultCheckoutQueries.cs]: "apple_pay" needs to go before "some_utf8_data_string"

**Comment by richardmonette** [comment]
Updated description: So it turns out the issue was that I was on the latest version of https://github.com/Shopify/graphql_schema, a gem we are using that is involved in processing the graphql_schema.json. Specifically, between the old version (0.1.3) and the new version (0.1.6) there was a change where the sorting of the fields was removed https://github.com/Shopify/graphql_schema/pull/10 This PR updates our tests to work with the latest gem version.

**Comment by richardmonette** [comment]
@sleroux Yes, mostly its sublime automatically cleaning up trailing spaces on line endings.

**Comment by dengjeffrey** [comment]
I think we should be bumping the `Gemfile` dependency as well to graphql_schema (~> 0.1.4)` at least since `0.1.4` has breaking changes compared to `0.1.3` which had sorting. Currently a dependency of `0.1.0` seems kind of a lie.

**Comment by dengjeffrey** [comment]
Or better yet, if it does work on `0.1.0` we should exclude `0.1.3` since that was the version with sorting

**Comment by richardmonette** [comment]
@dengjeffrey How about, `gem "graphql_schema", "~>0.1", ">=0.1.4"`, which would mean, any version of 0.1.X, but not 0.2.X, as long as 0.1.X has X larger than .4

**Comment by dengjeffrey** [comment]
I don't see a problem with including 0.2.X, I think our only issue is `0.1.3`, perhaps: 
```
gem "graphql_schema", "~>0.1.0", "!=0.1.3"
```
That includes everything we had before but excludes the version that caused the cases to break

**Comment by richardmonette** [comment]
@dengjeffrey Updated!

**Pull request merged**